lms/envs/devstack.py

```diff
@@ -158,3 +145,17 @@ def should_show_debug_toolbar(_):
     "API_ACCESS_KEY": "BBBBBBBBBBBBBBBBBBBB",
     "API_SECRET_KEY": "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
 }
+
+
+#####################################################################
+# See if the developer has any local overrides.
+try:
+    from .private import *      # pylint: disable=wildcard-import
```

Comment:
I change the pylint comment in here because the quality test was failing when `import-error` was provided